etor/src/test/groovy/gov/hhs/cdc/trustedintermediary/etor/ruleengine/transformation/custom/CopyOrcOrderProviderToObrOrderProviderTest.groovy

```diff
@@ -235,18 +237,18 @@ class CopyOrcOrderProviderToObrOrderProviderTest extends Specification{
         assert codingSystem == null || codingSystem[0]?.code == expectedIdentifierTypeCode
     }
 
+    Extension getObr16Extension(serviceRequest) {
```

Comment:
Would it be useful to have `getObr16Extension` and some of the functions in this test in `HapiHelper` to be reused?

Reply:
Refactored the OBR-16 and ORC-12 getters into HapiHelper